README.md

```diff
@@ -1,2 +1,58 @@
-# registry-operator
-Operator for devfile registry
+# Devfile Registry Operator
+
+Devfile Registry operator repository that contains the operator for the DevfileRegistry Custom Resource. 
+
+## Issue Tracking
+
+Issue tracking repo: https://github.com/devfile/api with label area/registry
+
+## Running the controller in a cluster
+
+The controller can be deployed to a cluster provided you are logged in with cluster-admin credentials:
+
+```bash
+export IMG=johncollier/registry-operator:v0.0.1
```

Comment:
Should we point the operator image to quay.io?

Reply:
This one should be resolved, I review commit by commit, the latest commit already has the changes